### PR TITLE
Pranjal Mishra

### PR DESCRIPTION
**Pull request by Pranjalmishra30**
Added Profile Card.
Issue solved : https://github.com/dsc-iem/AppDev-Hacktoberfest/issues/1#issue-705096059

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -7,28 +7,28 @@ packages:
...
-    version: "1.1.0-nullsafety.2"
+    version: "1.1.0-nullsafety"
   charcode:
     dependency: transitive
     description:
       name: charcode
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.2.0-nullsafety"
+    version: "1.1.3"
```
Comment (imKashyap) [inline pubspec.lock:31]: @Pranjalmishra30 Why did you change pubspec.lock?

Comment (Pranjalmishra30) [reply pubspec.lock:31]: I didnt change it intentionally, it changed on its own. May be because my flutter and android studio version are different.

Comment (imKashyap) [reply pubspec.lock:31]: Alright! I checked it.  I don't think it would create any issue.

Comment (Pranjalmishra30) [reply pubspec.lock:31]: Thanks !
